doc/macro_utils_design.md

```diff
@@ -235,4 +235,11 @@ MU_DIFFERENT(0,1) => 2
 
 Internally `MU_DIFFERENT` proposes numbers starting with MU_COUNT_ARG..0 as candidates for the expression result.
 
-If one of these numbers is not equal to any of the arguments then `MU_DIFFERENT` will evaluate to this number.
\ No newline at end of file
+If one of these numbers is not equal to any of the arguments then `MU_DIFFERENT` will evaluate to this number.
+
+### STATIC_ASSERT
```

Comment:
>STATIC_ASSERT [](http://example.com/codeflow?start=4&length=13)

All things in macro utils are prefixed with MU_
So I think this should be MU_STATIC_ASSERT

Reply:
Here and everywhere

Reply:
done